biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -1014,246 +1037,178 @@ function constrain_water_contents(th_uncorr, delta, ft, k) result(th_corr)
     real(r8) :: th_corr                ! corrected water content
     !
     !------------------------------------------------------------------------
-    ths     = EDPftvarcon_inst%hydr_thetas_node(ft,porous_media(k))
-    thr     = ths * EDPftvarcon_inst%hydr_resid_node(ft,porous_media(k))
+    ths     = EDPftvarcon_inst%hydr_thetas_node(ft,pm_type)
+    thr     = ths * EDPftvarcon_inst%hydr_resid_node(ft,pm_type)
```

Comment:
I see, so lets make sure the meta-data makes it clear that hydr_resid_node is per m3 of maximum saturation, ie relative units, or relative to saturation.

Reply:
@rgknox , I think the current definition is in relative terms already. Thus it is OK as it is for me; however, it might be cleaner and not easier to be confused if we set hydr_resid_node the same unit as hydr_thetas_node.

Reply:
I like those units more too, will change